modules.d/03rescue/module-setup.sh

```diff
@@ -8,6 +8,8 @@ check() {
 
 # called by dracut
 install() {
+    export hostonly="no"
```

Comment:
Doesn't this override the `hostonly` variable even outside the module? It should probably be stored and then restored after the install.

Reply:
In any case, I don't see why would you set it here, as it has no effect AFAIK.

Reply:
I got this idea from https://github.com/dracut-ng/dracut-ng/blob/main/modules.d/80test/module-setup.sh#L26 .

I changed it to a rescue config file as we discussed at https://github.com/dracut-ng/dracut-ng/pull/569